tensorflow/g3doc/api_docs/python/functions_and_classes/shard4/tf.contrib.layers.batch_norm.md

```diff
@@ -12,12 +12,12 @@ Can be used as a normalizer function for conv2d and fully_connected.
 Note: When is_training is True the moving_mean and moving_variance need to be
 updated, by default the update_ops are placed in `tf.GraphKeys.UPDATE_OPS` so
 they need to be added as a dependency to the `train_op`, example:
-
+```
```

Comment:
This markdown file is automatically generated. Can you revert these changes and make corresponding changes at: tensorflow/contrib/layers/python/layers/layers.py